### PR TITLE
Updated path in affected components.

### PR DESCRIPTION
**Pull request by slpysiren**
Updated paths to exploits to reflect the intended exploit path.

### CODE REVIEW

**Review by bradleybogg — COMMENTED** [review]
@slpysiren Please validate that each component's external reference originally is found within the exploits dir.

Need to ensure that post installation they are copied.

Please comment once verified and I'll start testing.

- [x] Verify each tool will be moved once installed.

**Review by bradleybogg — COMMENTED** [review]
Changes look good awaiting confirmation.

File: src/components/Ffuf/FfufTool.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const description_userguide =
     "and directories. This can reveal vulnerabilities in web applications as well generally data-mine " +
     "and map out the target. Ffuf can even be used to brute force credentials used in web authentication.\n\n" +
     "For further information on ffuf: https://github.com/ffuf/ffufS\n\n" +
-    "Wordlist directory: /src-tauri/exploits/ffuf_wordlists/\n\n" +
+    "Wordlist directory: /usr/shar/ddt/ffuf_wordlists/\n\n" +
```
Comment (zkkok) [inline src/components/Ffuf/FfufTool.tsx:17]: Seems to be a spelling error. Should be share instead of "shar"

**Review by zkkok — CHANGES_REQUESTED** [review]

File: src/components/Ffuf/FfufTool.tsx (hunk 1 above)
Comment (slpysiren) [reply src/components/Ffuf/FfufTool.tsx:17]: Thanks for catching this! Fixed now.

**Comment by slpysiren** [comment]
> @slpysiren Please validate that each component's external reference originally is found within the exploits dir.
> 
> Need to ensure that post installation they are copied.
> 
> Please comment once verified and I'll start testing.
> 
>     * [ ]  Verify each tool will be moved once installed.

I have verified all but one, GoldenEye doesn't seem to be included in its folder. I have found the tool on GitHub and attempted to include its files however, the GoldenEye folder seems to be listed as a submodule and I'm unable to stage any file additions inside. I can confirm the path would be correct if or when this is fixed.

**Review by bradleybogg — APPROVED** [review]

**Review by zkkok — APPROVED** [review]
Well done, no issues with the changes

**Comment by slpysiren** [comment]
> Hi @slpysiren,
> 
> When creating your branch please use the convention we have specified. This is best completed by creating it from the github issue.
> 
> Kind Regards,
> 
> Bradley

Hi Bradley, 

I'm not certain how to verify this but I believe I didn't create the branch. Though I will be sure to keep this in mind in future.

Thanks.

**Pull request merged**